scripts/build_layers.sh

```diff
@@ -25,7 +25,7 @@ function docker_build_zip {
     # between different node runtimes.
     temp_dir=$(mktemp -d)
     docker build -t datadog-lambda-layer-node:$1 . --no-cache \
-        --build-arg image=node:$1-alpine
+        --build-arg image=node:$1-alpine --network host
```

Comment:
What does this do?

Reply:
Good question, I don't know and I'm not sure why this is here, I have no recollection of adding this. I'll remove it from the PR. fwiw it [seems](https://docs.docker.com/network/host/) to be a feature that is only supported on Linux hosts, not on Docker for Mac anyway.